src/test/javascript/spec/component/markdown-editor/fullscreen-command.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslateModule } from '@ngx-translate/core';
+
+import { AceEditorModule } from 'app/shared/markdown-editor/ace-editor/ace-editor.module';
+import { FullscreenCommand } from 'app/shared/markdown-editor/commands/fullscreen.command';
+import { MarkdownEditorComponent } from 'app/shared/markdown-editor/markdown-editor.component';
+import { ArtemisMarkdownEditorModule } from 'app/shared/markdown-editor/markdown-editor.module';
+import { ArtemisTestModule } from '../../test.module';
+import * as FullscreenUtil from 'app/shared/util/fullscreen.util';
+
+describe('FullscreenCommand', () => {
+    let comp: MarkdownEditorComponent;
+    let fixture: ComponentFixture<MarkdownEditorComponent>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule, TranslateModule.forRoot(), AceEditorModule, ArtemisMarkdownEditorModule],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(MarkdownEditorComponent);
+                comp = fixture.componentInstance;
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should enable fullscreen on execute', () => {
+        jest.spyOn(FullscreenUtil, 'enterFullscreen');
+        jest.spyOn(FullscreenUtil, 'isFullScreen').mockReturnValue(false);
+
+        const command = new FullscreenCommand();
+        comp.defaultCommands = [command];
+        fixture.detectChanges();
+        comp.ngAfterViewInit();
+
+        command.execute();
+        expect(FullscreenUtil.isFullScreen).toHaveBeenCalledTimes(1);
+        expect(FullscreenUtil.enterFullscreen).toHaveBeenCalledTimes(1);
```

Comment:
Tbh extracting 5 lines into their own function isn't very sensible, as it would only reduce the LOC by 1. (Note that the methods differ in exit vs. enter, so only the middle part could be extraced)